CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1407](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1407))
 - `opentelemetry-instrumentation-logging` Add `otelTraceSampled` to instrumetation-logging
   ([#1773](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1773))
+- `opentelemetry-instrumentation-asgi` Add `http.server.response.size` metric
```

Comment:
Please move this entry to the Unreleased section

Reply:
Sorry about that. Now it's updated. Thanks.